tests/core/basic20/task.yaml

```
@@ -34,3 +34,10 @@ execute: |
     # ensure that our the-tool (and thus our snap-bootstrap ran)
     echo "Check that we booted with the rebuilt initramfs in the kernel snap"
     test -e /writable/system-data/the-tool-ran
+
+    echo "The recovery key is available"
+    test -e /var/lib/snapd/device/fde/recovery-key
+    echo "... and has the expected size"
+    test "$(stat --printf=%s /var/lib/snapd/device/fde/recovery-key)" = "16"
+    echo "... and has the expected owner and permissions"
+    test "$(stat --printf='%u %a' /var/lib/snapd/device/fde/recovery-key)" = "0 600"
```

Ideally we would check `%u %g %a`

```suggestion
    test "$(stat --printf='%u:%g %a' /var/lib/snapd/device/fde/recovery-key)" = "0:0 600"
```